### PR TITLE
Adding ACL scale test with error catching

### PR DESCRIPTION
**Pull request by simone-dell**
### Description of PR

Fixes # (issue)

### Type of change
- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Approach
How did you do it?
acltb_scale.yml gathers CRM information.  Based on the free count returned, the test scales to fill up all the free space in the hardware.  When an error is thrown, the scale number is returned.
How did you verify/test it?
Local repo
Any platform specific information?
No
Supported testbed topology if it's a new test case?
t1, t1-lag

### CODE REVIEW

**Comment by simone-dell** [comment]
Please don't merge this yet.  This is part of a work in progress so we can figure out how to make this test platform agnostic, as CRM show command is sometimes different across platforms.

File: ansible/roles/test/vars/testcases.yml [hunk 1]
```diff
@@ -6,9 +6,16 @@ testcases:
...
+      filename: 1acltb_scale.yml
+      topologies: [t1, t1-lag, t1-64-lag]
+      required_vars:
+          ptf_host:
+          testbed_type:
+
     arp:
       filename: arpall.yml
-      topologies: [ptf32, ptf64, t1, t1-lag, t1-64-lag]
+      topologies: [ptf32, ptf64, t1, t1-lag, t1-64-lag]
```
Comment (qiluo-msft) [inline ansible/roles/test/vars/testcases.yml]: >  [](start = 55, length = 1)

Remove the blank?

Comment (simone-dell) [reply ansible/roles/test/vars/testcases.yml]: removed, thanks

File: ansible/roles/test/tasks/acltb_scale.yml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+      debug:
+        msg: "CRM module still shows some available entries while SAI_INSUFFICIENT_ERROR shown"
+      when: available_count != 0
+
+- always:
+    - name: Clean up ACL rules.
+      vars:
+        command_to_run: "acl-loader update full /tmp/acltb_test_rules-del.json"
+        errors_expected: false
+      include: roles/test/tasks/run_command_with_log_analyzer.yml
```
Comment (wangxin) [inline ansible/roles/test/tasks/acltb_scale.yml]: Could you consider checking remaining ACL entry count from CRM again after the ACL rules are deleted?

**Comment by wangxin** [comment]
This scenario is already covered by `test_stress_acl.py`. I am closing this PR for now.

**Pull request closed without merging**